documentation/docs/libraries/rust/how_to/_run_how_tos.md

```diff
@@ -3,35 +3,24 @@
 Before you can run the examples, please refer to the [Rust Getting Started guide](./../getting_started/rust) to install
 the library.
 
-### Clone the Repository
-
-To run the rust examples, you will first need to clone the repository. You can do so by running the following command:
-
-```bash
-git clone git@github.com:iotaledger/wallet.rs.git
-```
-
 ### Set Up Your .env file
 
-After you have [cloned the project](#clone-the-repository), you should:
-
-1. Move into the project directory by running the following command:
+The code examples use a `.env` file to store variables. You can download
+the [example file](https://github.com/iotaledger/wallet.rs/blob/develop/wallet/.env.example) or create a new one with
+the following variables:
 
-```bash
-cd wallet.rs
-```
-
-2. Create your `.env` file by making a copy of the `.env.example` file by running the following command:
-
-```bash
-cp .env.example .env
+```dotenv
+NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC="endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river"
+STRONGHOLD_PASSWORD="some_hopefully_secure_password"
+NODE_URL="http://localhost:14265"
+FAUCET_URL="http://localhost:8091/api/enqueue"
 ```
 
 You should update the `NODE_URL` and `FAUCET_URL` values to match the [Hornet node](#hornet-node) you want to use.
```

Comment:
Same here, `https://faucet.testnet.shimmer.network/api/enqueue` for shimmer testnet faucet